File: 0001 Code Chef/0015 AC please/implementation/0003 c#/AcPlease.cs
using System;

class AcPlease{
    static void Main(string[] args){
        string stringInput = Console.ReadLine();
        int temperature = Convert.ToInt32(stringInput);
        if(temperature > 30){
            Console.WriteLine("YES");
        }else{
            Console.WriteLine("NO");
        }
    }
}

File: 0001 Code Chef/0015 AC please/implementation/0004 java/AcPlease.java
import java.util.Scanner;

class AcPlease{
    public static void main(String[] args){
        Scanner console = new Scanner(System.in);
        String stringInput = console.nextLine();
        int temperature = Integer.parseInt(stringInput);
        if(temperature > 30){
            System.out.println("YES");
        }else{
            System.out.println("NO");
        }
    }
}